sdk/appconfig/azappconfig/README.md

```diff
@@ -0,0 +1,298 @@
+      "endpoint": "https://my-app-config.azconfig.io",
+      "id": "/subscriptions/.../resourceGroups/my-resource-group/providers/Microsoft.AppConfiguration/configurationStores/my-app-config",
+      "location": "westus2",
+      "name": "my-app-config",
+      "provisioningState": "Succeeded",
+      "resourceGroup": "my-resource-group",
+      "tags": {},
+      "type": "Microsoft.AppConfiguration/configurationStores"
+  }
+  ```
+
+  > The `"endpoint"` property is the `endpointUrl` used by [Client][appconfig_client_src]
+
+### Authenticate the client
+This document demonstrates using the connection string. However, [Client][appconfig_client_src] accepts any [azidentity][azure_identity] credential. See the [azidentity][azure_identity] documentation for more information about other credentials.
+
+#### Create a client
+Constructing the client requires your vault's connection string, which you can get from the Azure Portal.
+```Bash
+export APPCONFIGURATION_CONNECTION_STRING="Endpoint=https://my-app-config.azconfig.io;Id=...;Secret=..."
+```
+
+```go
+import (
+    "os"
+    "github.com/Azure/azure-sdk-for-go/sdk/appconfig/azappconfig"
+)
+
+client, err = azappconfig.NewClientFromConnectionString(os.Getenv("APPCONFIGURATION_CONNECTION_STRING"), nil)
+```
+
+Or, using Default Azure Credential from Azure Identity:
+
+```go
+import (
+    "github.com/Azure/azure-sdk-for-go/sdk/appconfig/azappconfig"
+    "github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+)
+
+credential, err := azidentity.NewDefaultAzureCredential(nil)
+
+client, err = azappconfig.NewClient("https://my-app-config.azconfig.io", credential, nil)
+```
+
+## Examples
+This section contains code snippets covering common tasks:
+* [Add a configuration setting](#add-a-configuration-setting "Add a configuration setting")
+* [Get a configuration setting](#get-a-configuration-setting "Get a configuration setting")
+* [Set a configuration setting](#set-a-configuration-setting "Set a configuration setting")
+* [Set a configuration setting read only](#set-a-configuration-setting-read-only "Set a configuration setting read only")
+* [List configuration setting revisions](#list-configuration-setting-revisions "List configuration setting revisions")
+* [Delete a configuration setting](#set-a-configuration-setting "Delete a configuration setting")
+
+### Add a configuration setting
+
+```go
+import (
+    "fmt"
+    "os"
+
+    "github.com/Azure/azure-sdk-for-go/sdk/appconfig/azappconfig"
+    "github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+)
+
+func ExampleAddConfigurationSetting() {
+    connectionString := os.Getenv("APPCONFIGURATION_CONNECTION_STRING")
+    client, err = azappconfig.NewClientFromConnectionString(connectionString, nil)
+    if err != nil {
+        panic(err)
+    }
+
+    // Create configuration setting
+    resp, err := client.AddSetting(context.TODO(), azappconfig.Setting{Key: to.StringPtr("key"), Label: to.StringPtr("label"), Value: to.StringPtr("value")}, nil)
```

Comment:
It would be nice to add some whitespace here, just to make it more clear you're setting three fields in a Setting:
(whatever gofmt ends up doing)

    resp, err := client.AddSetting(context.TODO(), 
      azappconfig.Setting{
        Key: to.StringPtr("key"), 
        Label: to.StringPtr("label"), 
        Value: to.StringPtr("value")
      }, nil)

Reply:
(there are a few spots like this, in case you decide you want to change it)

Reply:
Thank you, I updated the samples to include more spacing/better formatting using newlines.